cmo /update-monitoring-plugin-image.yaml

```diff
@@ -0,0 +1,21 @@
+# 1) scale down Cluster Version Operator deployment to 0
+
+oc scale --replicas=0 -n openshift-cluster-version deployment/cluster-version-operator
```

Comment:
Why do we need to scale down the CVO?

Reply:
My understanding is that the CVO manages the Cluster monitoring operator. So, if we don't scale it down the CVO overwrites our changes when we modify the yaml `--image=monitoring-plugin=<foo-test-image>`. 

This is the advice I got from #forum-openshift-monitoring: https://redhat-internal.slack.com/archives/C0VMT03S5/p1719386516927169?thread_ts=1719350817.472789&cid=C0VMT03S5

Reply:
I believe the patch to CVO also prevents it from reconciling the CMO state, so we should only need to either scale it down or patch. I've only been patching it for the past couple of months and haven't had it restart CMO after I've patched it and scaled CMO down. As the version performs a lot of extra work I think it would likely be better to just perform the patch, but I could be wrong here.

Reply:
@PeterYurkovich So you're saying 
Either `oc scale --replicas=0 -n openshift-cluster-version deployment/cluster-version-operator`
OR this 
`oc patch clusterversion version --type json -p "$(cat disable-monitoring.yaml)"

oc scale --replicas=0 -n openshift-monitoring deployment/cluster-monitoring-operator

oc scale --replicas=0 -n openshift-monitoring deployment/monitoring-plugin
`
But not both.

Reply:
> So you're saying use either
> oc scale --replicas=0 -n openshift-cluster-version deployment/cluster-version-operator
> 
> OR this
> 
> oc patch clusterversion version --type json -p "$(cat disable-monitoring.yaml)"
> oc scale --replicas=0 -n openshift-monitoring deployment/cluster-monitoring-operator
> oc scale --replicas=0 -n openshift-monitoring deployment/monitoring-plugin

You should use either:
```bash
oc patch clusterversion version --type json -p "$(cat disable-monitoring.yaml)"
```
or
```bash
oc scale --replicas=0 -n openshift-cluster-version deployment/cluster-version-operator
```
Both of these actions prevent the clusterversion from fixing the CMO deployment. I would think the patch to be better since the rest of the clusterversion stays active in that case. 

```bash
oc scale --replicas=0 -n openshift-monitoring deployment/cluster-monitoring-operator
oc scale --replicas=0 -n openshift-monitoring deployment/monitoring-plugin
```
These are required for both